### ISSUE TITLE
[Tracking] API Progress

### ISSUE THREAD

**Issue by PhlexPlexico** [PhlexPlexico/G5API#1]
- [x] Users
- [x] Teams
- [x] Matches
- [x] API-Plugin
- [x] Leaderboard
- [x] Seasons
- [x] Game Server
- [x] Map Stats
- [x] Player Stats
- [x] Veto

**Comment by PhlexPlexico** [comment]
Match now has a rudimentary implementation of match forfeits. I'm not even sure if it works at this point as I have not tested it.

**Comment by PhlexPlexico** [comment]
Okay, all database calls should be ported over. The next thing will be getting Rcon calls, and checking if a server is available to send different calls. Everything is moving forward pretty smoothly now.

**Comment by PhlexPlexico** [comment]
Closing this as we have indeed finished the basic API operations. The next step is to start integrating the server calls, and perhaps replacing the legacy API with possible forwards to the new URL schema, and sending over the body? That way it would be possible to use this with a legacy plugin, and develop a new one that uses new calls. Maybe get the steam-passport to check if there's an API key being sent, instead of checking user-context as well. More on that when we get there.
